### ISSUE TITLE
Usage of keywords in namespace throws errors

### ISSUE THREAD

**Issue by nymo** [squizlabs/PHP_CodeSniffer#1952]
I have a namespace for example Tests\Unit\Global\Helper. I receive the error that PHP keywords must be lowercase; expected "global"but found "Global".

Since the code is being executed I don't see the point of throwing that kind of error. I know that the word "global" is reserved but since I use the uppercase version this shouldn't be a problem.

**Comment by jrfnl** [comment]
> I know that the word "global" is reserved but since I use the uppercase version this shouldn't be a problem.

Reserved keywords in PHP are case-*in*sensitive, so that argument holds no water.

**Comment by gsherwood** [comment]
If you have a namespace `Tests\Unit\Global\Helper` you're going to get a parse error because even PHP thinks that's a T_GLOBAL token. How is your code being executed?

**Comment by nymo** [comment]
@jrfnl thanks for clarification didn't know that they are case-insensitive.

@gsherwood you're right this throws an error. I was able to execute it because I defined the namespace first and run the code. Then in a next step I defined a "use Tests\Unit\Global\Helper...." and then the execution stopped. PHP is a little inconsistent at this case and should stop executing the code also when you use a keyword in a namespace. Preferably PHP should distinguish between normal usage of keywords and keyword like names in namespace and use statements.  But that's not the problem of this library.
